account_invoice_move_currency/models/account_invoice.py

```diff
@@ -45,27 +44,32 @@ def _compute_residual(self):
         Modificamos para que no lo convierta y muestre deuda en pesos si hay
         secondary currency
         """
-        if not self.move_currency_id:
-            return super(AccountInvoice, self)._compute_residual()
+        for rec in self:
+            if not rec.move_currency_id:
+                return super(AccountInvoice, self)._compute_residual()
```

Comment:
should be a super without return
and add a new line after that one with continue statement.
This in order to be able to properly compute the value for each value